examples/a3c/driver.py

```diff
@@ -10,16 +10,17 @@
 import six.moves.queue as queue
 import gym
 import sys
+import os
 from datetime import datetime, timedelta
-from misc import timestamp, Profiler
+from misc import *
```

Comment:
Can we avoid `import *`? I'd like to make this code PEP8 compliant later and the `import *` will make things a bit harder.